centreon/src/Core/Security/Token/Infrastructure/Repository/DbReadTokenRepository.php

```diff
@@ -164,6 +164,35 @@ public function existsByNameAndUserId(string $tokenName, int $userId): bool
         return (bool) $statement->fetchColumn();
     }
 
+    /**
+     * @param string $token
+     *
+     * @return bool
+     */
+    public function isTokenTypeManual(string $token): bool
+    {
+        try {
+            $statement = $this->db->prepare(
+            $this->translateDbName(
+                'SELECT token_type FROM `:db`.security_authentication_tokens WHERE token = :token'
+            )
+            );
+
+            $statement->bindValue(':token', $token, \PDO::PARAM_STR);
```

Comment:
bindValue() is useless for the type string because by default execute() apply binValue() with the string type 😄

Reply:
ok